cmd/zdb/zdb.c

```diff
@@ -2482,6 +2484,9 @@ dump_znode(objset_t *os, uint64_t object, void *data, size_t size)
 			    "path not found, possibly leaked");
 		}
 		(void) printf("\tpath	%s\n", path);
+		if (sa_lookup(hdl, sa_attr_table[ZPL_SYMLINK], &linktarget,
+		    sizeof (uint64_t)) == 0)
+			(void) printf("\ttarget	%s\n", linktarget);
```

Comment:
I think it would be useful to always print the symlink target when the object is a symbolic link (not just for -ddddd).  You can use with the `mode` returned above to determine if the object is a symbolic link and only then perform the `sa_lookup`.  Something like this:

```c
if (S_ISLNK(mode) && sa_lookup(hdl, sa_attr_table[ZPL_SYMLINK],
    &linktarget, sizeof (uint64_t)) == 0) {
        (void) printf("\ttarget	%s\n", linktarget);
}
```

Reply:
Done and tested.